### DISCUSSION TITLE
How to see scores with league.box_score

### DISCUSSION THREAD

**Discussion by wmblack23** [cwendt94/espn-api#240] [ESPN API] [answered]
Hello!  Super new to this API.

When I call league.box_sxore(1) it correctly shows me the matchups for week 1, but scores are not included.

Is there a way to include scores?

**Comment by cwendt94** [comment, marked as answer]
Hi @wmblack23, the scores should be there. [Box Score Class](https://github.com/cwendt94/espn-api/wiki/Box-Score-Class) is returned and you should be able to get the scores by looking at `home_score` and `away_score` variables.

**Reply by wmblack23** [reply]
Thanks
